CHANGELOG.md

```diff
@@ -1,5 +1,38 @@
 # Mattermost Mobile Apps Changelog
 
+## 1.28.0 Release
+- Release Date: February 16, 2020
+- Server Versions Supported: Server v5.9+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Compatibility
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
+ - iPhone 5s devices and later with iOS 11+ is required.
+ 
+### Highlights
+
+#### UI/UX Improvements to the Post Draft Area
+
+#### Deep Linking
+ - Links to posts in email notifications now launch to a browser landing page with option to open in the Desktop or Mobile app.
+
+### Improvements
+ - Added a post menu item to mark posts as unread.
```

Comment:
I think this is not part of this version but an older one, or am I wrong?

Reply:
The ticket and PR are marked for v1.28:
 - https://github.com/mattermost/mattermost-mobile/pull/3202
 - https://mattermost.atlassian.net/browse/MM-17068

Reply:
They are definitely not for this release. This was v1.26